tests/cli/test_cli.py

```diff
@@ -2366,6 +2366,12 @@ def test_nonexistent(ctx):
     _run_command([cl, 'work', 'nonexistent::'], expected_exit_code=1)
 
 
+@TestModule.register('rm_empty')
```

Comment:
This test is probably not running right now. You have to register the module in `test.yml`. Can you just add this test to the existing 'rm' module?